eng/pipelines/runtimelab/runtimelab-post-build-steps.yml

```diff
@@ -0,0 +1,22 @@
+steps:
+  # Zip CoreCLR Build Output
+  - template: /eng/pipelines/common/upload-artifact-step.yml
+    parameters:
+      rootFolder: $(Build.SourcesDirectory)/artifacts/bin/coreclr/$(osGroup).$(archType).$(buildConfigUpper)
+      archiveType: $(archiveType)
+      tarCompression: $(tarCompression)
+      includeRootFolder: false
+      archiveExtension: $(archiveExtension)
+      artifactName: CoreCLRProduct__$(osGroup)$(osSubgroup)_$(archType)_$(buildConfig)
+      displayName: 'CoreCLR product build'
+
+  # Zip Libraries Build Output
+  - template: /eng/pipelines/common/upload-artifact-step.yml
+    parameters:
+      rootFolder: $(Build.ArtifactStagingDirectory)/artifacts
```

Comment:
This will include all artifacts, coreclr, and installer as well, which is pretty big. I suggest you copying what you need for running tests:
For example we do this in runtime: https://github.com/dotnet/runtime/blob/master/eng/pipelines/libraries/build-job.yml#L101-L123

That will save build time and avoid being throttled by azdo, etc.

Reply:
good point, actually I copied this part from runtime so I'm actually using ArtifactStagingDirectory as well, the problem is that I didn't prepare that directory with the things I'm interested in which is why the build is failing, I'll fix this.